frontend/console/src/components/Divider.tsx

```diff
@@ -0,0 +1,10 @@
+interface DividerProps {
+  orientation?: 'horizontal' | 'vertical'
+  className?: string
+}
+
+export const Divider: React.FC<DividerProps> = ({ orientation = 'horizontal', className }) => {
```

Comment:
What do you think about making a prop `vertical?: boolean`? Then we don't have to worry about string typos not getting caught